README.md

```diff
@@ -31,9 +44,41 @@ For more detailed information about `mender-convert` please visit the
 [Debian family](https://docs.mender.io/artifacts/debian-family) section in
 [the Mender documentation](https://docs.mender.io/).
 
+## Included configurations
+
+### Core configurations
+
+These configurations are officially supported.
+
+| Configuration | Supported OS / hardware |
+| :------------ | :---------------------- |
+| beaglebone_black_debian_emmc_config | BeagleBone Black 4GB, Debian 32bit on internal eMMC storage |
+| beaglebone_black_debian_sdcard_config | BeagleBone Black, Debian 32bit on external SD card storage |
```

Comment:
I would put Beaglebone under contributed. It's true that we added it, and we are testing it during releases, but it's only because of self-interest: We want to verify that the UEFI code path for ARM works on at least one physical board. We have not successfully converted Raspberry Pi to UEFI yet, but it's looking increasingly likely we will do this soon, and then we will likely kill all efforts for Beaglebone. Hence better to avoid the term supported.